### ISSUE TITLE
Flaw: megaissue - часть изменений с апстрима потеряна

### ISSUE THREAD

**Issue by pheenty** [Space-Stories/space-station-14#249]
См. #123 #185
Такая же проблема, и не только с ограничениями времени, есть и в других местах.
Например:
У пассажира нет доступа в техи (он должен быть)
У basemobspecies всё ещё есть компонент fingerprint, который был перенесен в basemobspeciesorganic (а у нас он и там, и там. В итоге у скелета всё ещё есть отпечатки)
Снова ограничения времени, для ГСБ не нужно отыгрывать на вардене (а должно быть нужно)
И это 100% далеко не всё.

**Comment by doublechest0** [comment]
У пассажира нет доступа в техи (он должен быть) скорее всего с корвакса

**Comment by pheenty** [comment]
> У пассажира нет доступа в техи (он должен быть) скорее всего с корвакса

Да, так и есть. Сегодня пройдусь по всем профессиям и перенесу потерянное от оффов
